check_install.py

```diff
@@ -64,6 +61,8 @@ def main():
     p.expect([cmd_prompt])
     p.sendline('show ip bgp sum')
     p.expect([cmd_prompt])
+    p.sendline('sudo rm -r /host/image-*/rw/')
```

Comment:
If you think there is garbage in this folder, could you delete during building the image. Is it a issue on non-KVM hardware?

Reply:
Actually, that command isn't needed anymore with the changes in my second commit. Let me remove that.

The files in that folder comes when kvm boots up the SONiC image to verify basic functionality is working, not when the image is built. One type of file that's written into that folder under `/var/lib/systemd/timers` is timer stamp files, saying when was the last time a timer was started or was fired. The presence of these files affects the fstrim and logrotate timers when the image is actually deployed into a KVM/QEMU environment. It'll cause systemd to immediately fire the timers when they are started instead of waiting till whenever the next "event" is supposed to be, because systemd will think that based on the stamp files, these timers should have already ran.

This is not an issue on non-KVM images, because they are not booted up during image build; their first bootup would be on the hardware.